budgetApp1/budgetApp1/Repository.swift

```diff
@@ -26,18 +26,18 @@ protocol TransactionRepositoryType: AnyObject {
 class BudgetRepository: BudgetRepositoryType {
     
     func getBudgets(completion: @escaping (BudgetsResult)) {
```

Comment:
please dont use the word get. use fetch. get is very java, fetch is more swift😄

Reply:
Done